meetings/2024-01-17.md

```diff
@@ -0,0 +1,124 @@
+    terms of using the package manager that they.
+  * Michael, one option not discussed too much, mark stable but still require corepack enable,
+    * Mael might be ok, but what is the point of forcing the extra command
+    * Michael to try to make sure they know they are enable auto install of software for them
+  * Darcy - link that shows registry traffic
+    * Sharing some data sourced from the public npm team statements: <https://github-production-user-asset-6210df.s3.amazonaws.com/459713/297423226-1d18f2dc-1deb-457b-8a14-5f512921f8cc.png>
+    * there was almost a 50/50 split at one point, which shows that blockers of consumption might
+      not have been there original
+  * Steven - data shows us that people are using different package managers, the problem was
+    not that. The problem that corepack is solving is helping a new user get up and running
+    Faster.
+* Steven to answer the question about corepack by default. Seems like it should be easy.
+  Because it’s not enable by default Vercel does not have it enabled by default
+  * the official [@action/setup-node](https://github.com/actions/setup-node) where it will not be enabled because it is not enabled by default. There have been [multiple PRs](https://github.com/actions/setup-node/pulls?q=is%3Apr+corepack) to add support for corepack but still waiting on stable.
+  * value is that people who don’t even know they are using it, will use it when contributing to a new project.
+  * avoids the issue with multiple ways to install causing broken `PATH` - see  [example](https://twitter.com/galstar/status/1745172635674169663)
+* Claudio may be bad example but in terms of how many people use IE simply because its the
+  Default. More inclined if adopted by default.  Data might be different if others were enabled by
+  Default.
+* Darcy, only going to be aware of package managers that is aware of. Don’t think
+  its going to help with creating opportunities for package managers, more see
+  it as locking in the existing ones.  The 50/50 at one point supports this
+* Yagiz, disagree with the lock-in, not enabling the same level of competitive advantage is a
+  main concern for me. If all of them were shipped at the same time, then numbers might be
+  Different.  Alternative to enabling corepack would be to install all of them.
+* Mael, disagree that it would be locking in the package manager list. Any other package
+  manager could advocate to being added to the list.  Not being shipped as part of Node.js has
+  caused lots of issues over time.
+* Geoffrey, concern is that we ship both npm and corepack, npm taking firm stand on not
+  including npm, if its enabled by default, then people will ask why it’s not included.  Could we
+  add to npm, check of package manager, and spit out error saying using the wrong one.
+* Matteo, want to state the obvious, npm is the default, and it’s the golden standard, it is
```

Comment:
@benjamingr could you make a suggested chagnes to reflect what you think we should have in the minutes.

Reply:
```suggestion
* Matteo, want to state the obvious, npm is the default, and it’s the golden 
standard since all other package managers are npm registry clients. It is, 
```